Wasap/Wasap/DIContainer/AppDIContainer.swift

```diff
@@ -5,9 +5,57 @@
 //  Created by chongin on 9/29/24.
 //
 
+import UIKit
+
 final public class AppDIContainer {
     let apiClient: APIClient
     init(apiClient: APIClient) {
         self.apiClient = apiClient
     }
+    
+    public func makeWifiAutoConnectDIContainer() -> WifiAutoConnectDIContainer {
+        return WifiAutoConnectDIContainer()
+    }
+}
+
+final public class WifiAutoConnectDIContainer {
+    private var wifiConnectRepository: WiFiConnectRepository?
```

Comment:
이거 만들 때 바로 다음과 같이 해주면 될 것 같아용
```suggestion
    private var wifiConnectRepository = DefaultWiFiConnectRepository()
```